server/paymentProviders/transferwise/index.ts

```diff
@@ -788,25 +789,41 @@ const oauth = {
   },
 };
 
-async function setUpWebhook(): Promise<Webhook> {
-  const url = `${config.host.api}/webhooks/transferwise`;
+const APPLICATION_TRIGGERS = ['transfers#state-change', 'transfers#refund'] as const;
+
+async function createWebhooksForHost(host = config.host.api): Promise<Webhook> {
```

Comment:
I don't see where we're using this function?

Depending on how we use it, `host` seems like a dangerous parameter. Why not just always use `config.host.api` , eventually mocking the config in tests if it what it's for?

Reply:
There's a part of this PR that was stashed and contained a helper tool to mount and dismount webhooks using ngrok.

But you're right, we can lock this to `config.host.api`.